tests/test-lib-proxy-ip-forward.php

```diff
@@ -251,6 +329,29 @@ public function test__defined() {
 	}
 }
 
+class IP_Forward__Is_Valid_Proxy_Verification_Key__Test extends \PHPUnit_Framework_TestCase {
```

Comment:
For these to work, we'll either need to run each test in a separate process (see `IP_Forward__Get_Proxy_Verification_Key__Test` and define the constant for each) or mock the return value of `get_proxy_verification_key()`